Library/Dnn.PersonaBar.UI/admin/personaBar/scripts/permissionGrid.js

```diff
@@ -7,6 +7,17 @@ if (typeof dnn === "undefined" || dnn === null) { dnn = {}; }
 if (typeof dnn.controls === "undefined" || dnn.controls === null) { dnn.controls = {}; }
 
 (function ($) {
+	
+	var permissionsMap = {
+		BROWSE_FOLDER: 8,
+		VIEW_FOLDER: 5,

```

Comment:
Given that these are defined in the database, I'd be more comfortable loading these IDs from the server, rather than assuming they're always these hardcoded values (unless I'm misunderstanding and these are also hardcoded elsewhere)

Reply:
@mean2me do you know how this is handled in other places in code? A `GET` endpoint to retrieve these values does make sense. Especially that you introduced the exact same thing in #83